pom.xml

```diff
@@ -32,6 +32,12 @@
             <version>5.5.2</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>commons-cli</groupId>
+            <artifactId>commons-cli</artifactId>
+            <version>1.5.0</version>
```

Comment:
Please, move version to `<properties/>` block.
And if it's not so hard, do it for other dependencies